tasks/client_keys.yml

```diff
@@ -1,94 +1,95 @@
-# generate client keys
 ---
-- set_fact:
+- name: Set facts for additional certs
+  set_fact:
     additional_certs_to_revoke: "{{ _additional_certs_to_revoke.stdout_lines | default([]) }}"
     additional_certs_to_add: "{{ _additional_certs_to_add.stdout_lines | default([]) }}"
 
 - name: Create openvpn ovpn file directory
   file:
-    path: "{{openvpn_ovpn_dir}}"
+    path: "{{ openvpn_ovpn_dir }}"
     state: directory
+    mode: "0755"
 
 - name: Copy openssl client extensions
   copy:
     src: openssl-client.ext
-    dest: "{{openvpn_key_dir}}"
+    dest: "{{ openvpn_key_dir }}"
     owner: root
     group: root
     mode: 0400
 
-- debug: msg={{ additional_certs_to_add }}
-
-- name: generate client key
-  command: "openssl req -nodes -newkey rsa:{{openvpn_rsa_bits}} -keyout {{item}}.key -out {{item}}.csr -days 3650 -subj /CN=OpenVPN-Client-{{ inventory_hostname[:24] }}-{{ item[:24] }}/"
+- name: Generate client key
+  command: >-
```

Comment:
Whoa. I didn't know about the `>-` yaml syntax

Reply:
Yes it's called the "block chomping operator" :D https://yaml.org/spec/1.2/spec.html#id2794534